admin/create-dumps.sh

```diff
@@ -77,6 +77,9 @@ if [ -z $DUMP_BASE_DIR ]; then
 fi
 
 DUMP_TYPE="${1:-full}"
+# consume dump type argument so that we can pass the remaining arguments to
+# the python dump manager script
```

Comment:
in this case it'd be good to have some kind of documentation for this script that explains that we're able to do this. Perhaps just comments at the top of the file like we do in develop.sh and test.,sh is enough for now

Reply:
Added a few line on script usage at top.